samples/start-app/rust-actix/src/configuration.rs

```diff
@@ -0,0 +1,65 @@
+use std::{env, fs};
+
+#[derive(Debug, Clone)]
+pub struct Settings {
+    pub app_server_addr: String,
+    pub pim_url: String,
+    pub client_id: String,
+    pub client_secret: String,
+    pub scopes: String,
+    pub secure_cookie: bool,
+    pub session_key: String,
+    pub app_name: String,
+    pub log_level: String,
+    pub sub_hash_key: String,
+}
+
+// https://api.akeneo.com/apps/authentication-and-authorization.html#authorization-and-authentication-scopes
+static PIM_AUTHORIZATION_SCOPES: &str =
+    "openid email profile read_channel_localization read_channel_settings";
+
+impl Settings {
+    pub fn get(path: Option<String>) -> Self {
+        self::Settings::load_env_var(path);
+
+        let app_host = env::var("APP_HOST").expect("APP_HOST is not set in .env file");
+        let app_port = env::var("APP_PORT").expect("APP_PORT is not set in .env file");
+        let session_key = env::var("SESSION_KEY").expect("SESSION_KEY is not set in .env file");
```

Comment:
technically it misses an env var, doesn't have to come from .env file.

Reply:
True, messages are updated.